webapp/src/components/MortgageActions/MortgageActions.container.js

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux'
+import MortgageActions from './MortgageActions'
+import { cancelMortgageRequest } from 'modules/mortgage/actions'
+
+const mapState = (state, ownProps) => {
+  const { mortgage } = ownProps
+  return {
+    mortgage
+  }
+}
+
+const mapDispatch = (dispatch, { mortgage }) => ({
+  onCancel: () =>
+    dispatch(
+      cancelMortgageRequest(mortgage.mortgage_id, {
+        assetId: mortgage.asset_id
+      })
```

Comment:
I think the signature for cancelMortgageRequest is `cancelMortgageRequest(mortgageId, assetId)` so it should be `cancelMortgageRequest(mortgage.mortgage_id, mortgage.asset_id)`

Reply:
☝️